### PR TITLE
javascript: Providing documentation and examples for Temporal types

### PR DESCRIPTION
**Pull request by bigmontz**
Adding section with usage examples of the temporal types.

A sub-section is added to https://neo4j.com/docs/javascript-manual/current/cypher-workflow/#js-driver-type-mapping

See: 
Screenshot 2021-06-07 at 15 10 36

### CODE REVIEW

**Review by gjmwoods — APPROVED** [review]
🕐

**Comment by davidoliverSP2** [comment]
Hi @bigmontz - I'm having trouble building this with the new temporal examples. Have those examples been included in a recent release of the JS Driver?

**Comment by bigmontz** [comment]
> Hi @bigmontz - I'm having trouble building this with the new temporal examples. Have those examples been included in a recent release of the JS Driver?

It's merged to the 4.2 and 4.3 branches of the driver. Should the driver be released ?

**Comment by davidoliverSP2** [comment]
> > Hi @bigmontz - I'm having trouble building this with the new temporal examples. Have those examples been included in a recent release of the JS Driver?
> 
> It's merged to the 4.2 and 4.3 branches of the driver. Should the driver be released ?

Thanks @bigmontz - we've been looking into this. Currently it seems that we reply on _released_ drivers for the examples, so we wouldn't expect them to render in the current PR setup. 

@recrwplay and I are going to look into whether we want to change this arrangement.

File: jsManual/asciidoc/cypher-workflow.adoc [hunk 1]
```diff
@@ -127,6 +127,64 @@ include::{common-content}/cypher-workflow.adoc[tag=type-mapping]
...
+Represents an instant capturing the time of day, but not the date, nor the timezone.
+
+[source, javascript]
+----
+include::{javascript-examples}/examples.test.js[tags=temporal-types-localtime]
+----
+
+[[js-driver-temporal-types-duration]]
+==== Duration
+Represents an ISO 8601 duration. Contains both date-based values (years, months, days) and time-based values (seconds, nanoseconds).
```
Comment (davidoliverSP2) [inline jsManual/asciidoc/cypher-workflow.adoc]:
```suggestion
Represents an ISO 8601 duration. 
Contains both date-based values (years, months, days) and time-based values (seconds, nanoseconds).
```

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @bigmontz - please could you let me know when these examples are in a released version of the Driver, and we can update the published docs.

**Pull request merged**